repository/src/main/java/com/epam/esm/ErrorType.java

```diff
@@ -0,0 +1,6 @@
+package com.epam.esm;
+
+public enum ErrorType {
```

Comment:
Are you going to put error codes here for other exceptions? The name of the enumeration looks rather abstract

Reply:
fixed to more specific